src/server.js

```diff
@@ -148,6 +148,12 @@ module.exports = function (options = {}) {
 	server.listen(listenParams, () => {
 		if (typeof listenParams === "string") {
 			log.info("Available on socket " + colors.green(listenParams));
+			fs.chmod(listenParams, Helper.config.socketPermissions, (err) => {
+				if (err) {
+					log.error(`fs chmod error: ${err}. Stopping server...`);
```

Comment:
s/fs/socket/

That it is using the fs module is really not relevant, what it tried to chmod is.
Bonus points if you also add the path, assuming that it may not be in the error message itself, depending on what OS and what error you get.